tests/test_client/tests.py

```diff
@@ -1250,6 +1251,15 @@ async def test_query_params(self):
                 )
                 self.assertEqual(response.asgi_request.GET["example"], "data")
 
+    async def test_async_header_defaults(self):
+        async_client = AsyncClient(HTTP_AUTHORIZATION="Bearer I_AM_JWT_TOKEN")
+
+        response = await async_client.get(
+            "/async_default_headers/",
+        )
+
+        self.assertEqual(response.content, b'{"message": "This is a trial view"}')
```

Comment:
When I revert the changes this test passes
Instead of this (and adding the new view and url), perhaps:
```diff
--- a/tests/test_client/tests.py
+++ b/tests/test_client/tests.py
@@ -1337,6 +1337,18 @@ class AsyncRequestFactoryTest(SimpleTestCase):
         self.assertEqual(request.headers["x-another-header"], "some other value")
         self.assertIn("HTTP_X_ANOTHER_HEADER", request.META)
 
+        request_factory_with_headers = AsyncRequestFactory(
+            **{
+                "Authorization": "Bearer faketoken",
+                "X-Another-Header": "some other value",
+            }
+        )
+        request = request_factory_with_headers.get("/somewhere/")
+        self.assertEqual(request.headers["authorization"], "Bearer faketoken")
+        self.assertIn("HTTP_AUTHORIZATION", request.META)
+        self.assertEqual(request.headers["x-another-header"], "some other value")
+        self.assertIn("HTTP_X_ANOTHER_HEADER", request.META)
+
     def test_request_factory_query_string(self):
```